README.md

```diff
@@ -74,6 +75,7 @@ No Wayland-specific requirements, so you can use your xorg solution of choice to
 
 ## Notifications
 
+* [dunst](https://dunst-project.org/) - Dunst is a lightweight replacement for the notification daemons provided by most desktop environments
```

Comment:
Did you add Dunst, because it works via XWayland normally?

Reply:
Sorry, there is a wayland branch, but for the moment, you can delete dunst.

Reply:
```suggestion
```

Sounds good! Thanks PRESFIL for the catch, and vperilla for the additions!